pangeo-notebook/packages.txt

```diff
@@ -677,63 +688,65 @@ unicodedata2==15.1.0
 uri-template==1.3.0
 uriparser==0.9.8
 urllib3==1.26.19
-uvicorn==0.32.0
-uvicorn-standard==0.32.0
+uvicorn==0.34.0
+uvicorn-standard==0.34.0
 uvloop==0.21.0
-watchfiles==0.24.0
+watchfiles==1.0.3
 watermark==2.5.0
 wayland==1.23.1
 wayland-protocols==1.37
 wcwidth==0.2.13
-webcolors==24.8.0
+webcolors==24.11.1
 webencodings==0.5.1
 webob==1.8.9
 websocket-client==1.8.0
-websockets==13.1
-wheel==0.45.0
+websockets==14.1
+wheel==0.45.1
 widgetsnbextension==4.0.13
-wrapt==1.16.0
+wrapt==1.17.0
 x264==1!164.3095
 x265==3.5
-xarray==2024.10.0
+xarray==2025.1.1
```

Comment:
Fresh off the release, compatibility with Zarr v3! https://github.com/pydata/xarray/releases/tag/v2025.01.1